### ISSUE TITLE
Chemical Request

### ISSUE THREAD

**Issue by bethanyriggle** [afsc-gap-products/Survey-Planning#61]

**Comment by SarahFriedman-NOAA** [comment]
Just an update on this task progress for posterity: 

I've talked with Joanna Magner (the new point person in RSST for chemical requests) and she confirmed that the amounts for each chemical for 2021 were the following:
> - Ethanol: 20 gallons (Ocean Explorer), 30 gallons (AK Provider)
> - Formalin: 5 gallons (both boats)
> - Otolith solution: 15 gallons (both boats)

It seems that the extra 10 gallons of ethanol on the AKP were for a sponge project for Sean and I'm unsure if that is continuing this year. So, we'll likely be sticking with the same amounts from last year (minus the extra 10 gal. of ethanol), but I'm waiting for (1) Nate to confirm these values with Jay Orr and (2) to read over the special project requests this year and make sure there isn't anything that would require extra chemicals on board.

**Comment by SarahFriedman-NOAA** [comment]
I submitted the following request to Joanna, so I think we can probably close this issue. Just recording the amounts of each chemical here for posterity.

Ethanol: 30 gallons (Vessel 2), 30 gallons (AK Provider)
Formalin: 5 gallons (Vesssel 2), 8 gallons (AK Provider)
Otolith solution: 15 gallons (both boats)

I added the extra 10 gal of ethanol to both boats for Sean Rooney's special project this year at his request. The extra 3 gal of formalin on the AKP is for Christina's special project also at her request.
